src/test/java/com/camping/kiosk/steps/KioskSteps.java

```diff
@@ -20,14 +25,14 @@ public KioskSteps() {
 
     @When("키오스크 컨테이너에 요청을 보낸다")
```

Comment:
> 그리고 step 중 기능적으로는 동일하지만, Given 단계인지 When 단계인지에 따라 또 세부적으로 해줘야 하는 작업들이 달라지게 되면 애매하게 재사용하기 힘들 수도 있겠다는 생각도 드는 거 같아요. 현업에서 적용하시면서 이런 류의 고민은 어떤 식으로 해결하셨을지도 궁금하네요 ㅎㅎ...

이런 스텝의 경우 When이지만 Given도 될 수 있는 스텝인데요
하나의 메서드에 여러개의 애너테이션을 붙여서 지정해줄 수 있습니다!

```suggestion
    @Given("키오스크 컨테이너에 요청을 보냈다")
    @When("키오스크 컨테이너에 요청을 보낸다")
```